dpgen/auto_test/EOS.py

```diff
@@ -24,6 +24,8 @@ def __init__(self,
                 self.vol_start = parameter['vol_start']
                 self.vol_end = parameter['vol_end']
                 self.vol_step = parameter['vol_step']
+                parameter['vol_abs'] = parameter.get('vol_abs', False)
```

Comment:
[`set_default`](https://docs.python.org/3/library/stdtypes.html#dict.setdefault) is better than `get`.

Reply:
`get` is applied in other parts of dpgen. To keep consistency, I think it's ok.

Reply:
If you mean the other parts in this file, indeed I don't think they are ok. Here `set_default` is faster and more concise. I can approve this PR though.